test_e2e/test_app.py

```diff
@@ -0,0 +1,82 @@
+import os
+import app
+import pytest
+import trello_service
+from card import Status
+from dotenv import load_dotenv, find_dotenv
+from selenium import webdriver
+from threading import Thread
+
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Chrome('/usr/local/bin/chromedriver') as driver:
```

Comment:
This requires chromedriver to be somewhere specific, only works on Unix systems and is undocumented.

You can actually just do `webdriver.Chrome()` and it will work if it's on your path.